docs/docs/api/08-Configuration/askui-ui-controller.md

```diff
@@ -114,3 +114,14 @@ describe('jest with askui', () => {
   })
 })
 ```
+
+## Download
+
+Download the binary/binaries for the respective platform(s):
+
+* [Windows](https://files.askui.com/releases/askui-ui-controller/latest/win32/x64/askui-ui-controller.exe)
+* [macOS(intel)](https://files.askui.com/releases/askui-ui-controller/latest/darwin/x64/askui-ui-controller.dmg)
+* [macOS(silicon)](https://files.askui.com/releases/askui-ui-controller/latest/darwin/arm64/askui-ui-controller.dmg)
+* [Linux](https://files.askui.com/releases/askui-ui-controller/latest/linux/x64/askui-ui-controller.AppImage)
+
+Feeling adventurous? Try our new [Remote Device Controller (only Windows)](../09-Remote-Device-Controller.md).
```

Comment:
Not sure if this is really clear to the user as we talk about the UI Controller (and not Remote Device Controller) throughout the page and it is also name that way on the lib side. 

My idea would be to add it to the download binaries, e.g., something like


...
* [Windows](https://files.askui.com/releases/askui-ui-controller/latest/win32/x64/askui-ui-controller.exe)
* [Windows (new version - beta release)](<link to new exe or installer>) ([more about the new version](../09-Remote-Device-Controller.md))
* [macOS(intel)](https://files.askui.com/releases/askui-ui-controller/latest/darwin/x64/askui-ui-controller.dmg)
...

Reply:
I like this a lot!